meson_options.txt

```diff
@@ -13,6 +13,9 @@ option('with-kernel-modules-dir', type: 'string', description: 'System kernel mo
 option('with-kernel-namespace', type: 'string', description: 'Identifier for boot assets', value: 'org.clearlinux')
 option('with-kernel-vendor-conf-dir', type: 'string', description: 'Vendor kernel configuration directory')
 
+# Initrd options
+option('with-initrd-dir', type: 'string', description: 'Vendor initrd without kernel dependency directory')
```

Comment:
Terminology is a bit janky, perhaps "Freestanding initrd" is better. By definition initrd requires a kernel, as the kernel loads it. It's not tied to a **specific** kernel however, thus it's freestanding but not without dependency.

Reply:
totally agree! :D